login/service_whitebox_test.go

```diff
@@ -156,8 +156,7 @@ func TestEncodeTokenOK(t *testing.T) {
 	refreshToken := "refreshToken%@!/\\&?"
 	tokenType := "tokenType%@!/\\&?"
 	expiresIn := 1800
-	var refreshExpiresIn float64
-	refreshExpiresIn = 2.59e6
+	refreshExpiresIn := 2.59e6
```

Comment:
but I am wondering how `assert.Equal` returned `True`.
Is it verifying ONLY the value then?